package-lock.json

```diff
@@ -4,10 +4,82 @@
+      "integrity": "sha512-9lwrKso0XjyS7wu+8dEWa5yN1kCTdbelP6JElFhh0kAt0TbPVHJ/dXEwvIFk9/2MjYv2PbooQo1zsc5kAB2Rlg==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/compiler": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/compiler/-/compiler-8.0.2.tgz",
+      "integrity": "sha512-ktobrxpWX1eCwbDKOIUm5GRj8WGlHW/8MAQvDDFUnsGqXBHfOGiaySiEYX/XjeN8qu34IfXs736QkdzpMM4+iw==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/core": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/core/-/core-8.0.2.tgz",
+      "integrity": "sha512-g8BRvGZxTXb5GZ/xoC5Z94DGK3wMiD2jbmEQEbXGNM+c8E/Mo/W8GF44P7EU2d+V1oJoUh75SRK6U/StC+rLqA==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/forms": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/forms/-/forms-8.0.2.tgz",
+      "integrity": "sha512-LGu3b/wjNMCki5PnMUsfQlyaVZVOedNO+XccfluP4ZBQ5G/E2cz2tJ0UIHg3RhLbbpWntmqokpYLyd7leUPpIQ==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/platform-browser": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/platform-browser/-/platform-browser-8.0.2.tgz",
+      "integrity": "sha512-iUoyhJ81jqvpmQI6Lu5NzRZR8azmnb2kX2FQ+LbwCvWQLfkLbTaa/Jl09/qN6KWpTsMogNQXVnjjgwoeaObvBw==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/platform-browser-dynamic": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/platform-browser-dynamic/-/platform-browser-dynamic-8.0.2.tgz",
+      "integrity": "sha512-i9hBmcXauiQpvzRRNbBpMbQSsRh9YMv9phw9Ih54w2LdTUB64Vtrm3scQEWZ0sIONY3QbwEtu57k8euQPASErQ==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/router": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/router/-/router-8.0.2.tgz",
+      "integrity": "sha512-kgSqkvx3cgY0+ZTzkBVLa0sf0y0Fl7q3N+KX746pbCNjRRGa5gBkN7z+oyPDoG1l0fEiM6NLbFfcmuFwsdBO5w==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
+    "@angular/upgrade": {
+      "version": "8.0.2",
+      "resolved": "https://registry.npmjs.org/@angular/upgrade/-/upgrade-8.0.2.tgz",
+      "integrity": "sha512-ECsMGFTWjuoD++QMl7PUIax7YQ/UC589kqNhxW8zKkjiOqhd/NnZ+GxcnzsJtNklARpJ2DndXEEeIOQALOe3bA==",
+      "dev": true,
+      "requires": {
+        "tslib": "^1.9.0"
+      }
+    },
     "@babel/code-frame": {
       "version": "7.0.0",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
-      "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
+      "integrity": "sha1-BuKrGb21NThVWaq7W6WXKUgoAPg=",
```

Comment:
Try to regenerate the package-lock.json file again, we typically want SHA-512 hashes instead of SHA1 (which is considered to be a broken hash).

Reply:
The npm was really old (it did not even have npm audit!). I have updated my npm and it now has SHA-512 hashes

Reply:
Hmm, I still see SHA1 hashes, PTAL!

Reply:
I have added more SHA512 (I deleted the node_modules/ folder and also package-lock and then ran `npm i` to get a fresh package-lock) though there are many SHA1 in develop too.